design/account.go

```diff
@@ -94,6 +94,35 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	})
 })
 
+// userSpaces a list of spaces in which the user has a role
+var userSpaces = JSONList(
+	"userSpaces", "Holds the paginated response to a user spaces request",
+	userSpacesData,
+	pagingLinks,
+	userSpacesListMeta)
+
+// userSpacesData represents a resource for which a user has a role
+var userSpacesData = a.Type("UserSpacesData", func() {
+	a.Attribute("id", d.String, "id of the resource that in which the user has a role")
```

Comment:
It's not just some resource. It's always the ID of the space. IMO it's worth being more specific here.

Reply:
right, let me fix that.